### PR TITLE
Export disk metrics as part of `/metrics-export` endpoint

### LINKED ISSUE #994: Export new disk metrics via `/metric-export` endpoint.
The newly exposed disk usage metrics are required for GKE monitoring. `/metric-export` and `/metric-export-schema` REST endpoints should include pod volumes usage data and container's logs and write layer usage data, if available.

Essentially, [`LabeledMetrics`](https://github.com/kubernetes/heapster/blob/heapster-scalability/metrics/core/metrics.go#L66) should also be exported. 

This will let us expose disk usage for pods and containers in GKE.

cc @timstclair @mwielgus

### PR DESCRIPTION
**Pull request by vishh**
Fixes #994

### CODE REVIEW

**Comment by vishh** [comment]
@k8s-bot test this

File: Makefile [hunk 1]
```diff
@@ -4,11 +4,11 @@ TAG = v0.20.0-alpha8
 PREFIX = gcr.io/google_containers
 FLAGS = 
 
-SUPPORTED_KUBE_VERSIONS = "1.0.6"
+SUPPORTED_KUBE_VERSIONS = ""
```
Comment (mwielgus) [inline Makefile]: I guess something should remain here for manual runs.

Comment (vishh) [inline Makefile]: Really? IN its current state, it should run the e2e tests against an existing cluster.

**Comment by mwielgus** [comment]
LGTM

**Comment by mwielgus** [comment]
Fixes #934

**Pull request merged**